01-fundamentals/first_letter.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2016 Ami . All rights reserved.
+
+def get_user_password():
+    password = input("Tell me your password:")
+    return password
+
+
+def show_first_letter_of(password):
+    first_letter = get_first_letter_of(password)
+    print("The first letter you entered was: " + first_letter)
+
+def get_first_letter_of(text):
```

Comment:
Like above, the `_of` is not really needed. What's needed is some mentioning that this functions returns the letter in uppercase. Imagine what if you had an additional function that returns the first letter in lower case?